DataFormats/PatCandidates/interface/Muon.h

```diff
@@ -348,6 +390,20 @@ namespace pat {
 
       /// Muon MVA
       float mvaValue_;
+
+      /// MC matching information
+      MuonSimType simType_;
+      int simFlavour_;
+      int simHeaviestMotherFlavour_;
+      int simPdgId_;
+      int simMotherPdgId_;
+      int simBX_;
+      float simProdRho_;
+      float simProdZ_;
+      float simPt_;
+      float simEta_;
+      float simPhi_;
```

Comment:
It may be more useful to make a struct-like class for these e.g. in SimDataFormats/Associations
so that it can be used in other places without having to pull dependencies coming with pat::Muon.

Reply:
That's what @davidlange6 suggested as well. Working on it.

Reply:
done